src/main/java/com/bandwidth/sdk/messaging/MessagingClient.java

```diff
@@ -41,23 +45,84 @@
     private final AsyncHttpClient httpClient;
     private final MessageSerde messageSerde = new MessageSerde();
 
-    /**
-     * Credentials to access Bandwidth's Messaging V2 api
-     *
-     * @param userId Ex: u-1a2b3c4d
-     * @param token  Ex: t-1a2b3c4d
-     * @param secret Ex: a3947ouilar
-     */
-    public MessagingClient(String userId, String token, String secret) {
-        this.userId = userId;
+    public static MessagingClient.Builder builder() {
+        return new MessagingClient.Builder();
+    }
+
+    public static class Builder {
+
+        private static final String USER_AGENT_HEADER_VALUE = "messaging-java-sdk";
+
+        /**
+         * {@link RequestFilter} that adds the required "x-realm: admin" header to all outbound requests.
```

Comment:
This is the comment from a different `RequestFilter` that was required for the Numbers SDK. IMO it would be fine to just remove and leave uncommented, but if kept should be changed to say that this adds the `user-agent` header.